html-forms/index.html

```diff
@@ -21,21 +21,15 @@ <h1>Coding Form!</h1>
           <option>Because it's powerful</option>
         </select>
       </div>
-      <div>
-        <label for="language">Favorite Language</label>
-      </div>
-      <div>
-        <input type="radio" id="html" name="language" />
-        <label for="html">HTML</label>
-        <input type="radio" id="css" name="language" />
-        <label for="css">CSS</label>
-        <input type="radio" id="javascript" name="language" />
-        <label for="javascript">JavaScript</label>
-      </div>
-      <div>
-        <label for="sign"
-          >Would you like to sign up for more information?</label
-        >
+      <div>Favorite Language (Choose one)</div>
+      <input type="radio" id="html" name="language" />
+      <label for="html">HTML</label>
+      <input type="radio" id="css" name="language" />
+      <label for="css">CSS</label>
+      <input type="radio" id="javascript" name="language" />
+      <label for="javascript">JavaScript</label>
+      <div>
+        <div>Would you like to sign up for more information?</div>
```

Comment:
You shouldn't use `div` elements for text content, `div` elements are meant to be for layout control. I would use a `p` element.